rust/core-lib/src/editor.rs

```diff
@@ -423,7 +423,7 @@ impl Editor {
         let mut saved = None;
         for region in sel_regions {
             if !region.is_caret() {
-                let val = self.text.slice_to_string(region.min(), region.max());
+                let val = self.text.slice_to_string(region.min()..region.max());
```

Comment:
I wonder if we can just implement `RangeBounds` for `SelRegion`, and then we'll be able to pass it in to things like this directly?